### PR TITLE
Llmnr client

### PR DESCRIPTION
**Pull request by yrro**

### CODE REVIEW

File: config/services/llmnr.xml
Comment (erig0) [inline config/services/llmnr.xml:6]: What do you think about allowing `llmnr-client` here? I think a user enabling `llmnr` will also want to hear the replies.

Note that `ws-discovery` does NOT include `ws-discovery-client`, but maybe it should. I wonder how @gbsf and @marcosfrm feel about this.

Comment (marcosfrm) [reply config/services/llmnr.xml:6]: We should avoid `source-port` as much as possible. rfc4795 says implementations are free to discard unsolicited responses...

Comment (erig0) [reply config/services/llmnr.xml:6]: @marcosfrm, fair enough. Let's leave out the `-client` variants then.

Comment (gbsf) [reply config/services/llmnr.xml:6]: The thing is, while the `-client` services are a kludge, they are basically required to use on desktop machines. Until we can get conntrackd working, we have to rely on an implementation detail that Linux applications in general use the same request port number as the response, instead of a random one. The multicast-to-unicast dance leaves no other option, otherwise we end up discarding _solicited_ responses too…
If they are not enabled by default alongside the main service, I feel that at least the description of those should mention that `-client` is currently needed to receive replies.

Comment (erig0) [reply config/services/llmnr.xml:6]: Let me ask it this way.

Would a user _expect_ that enabling the `llmnr` service will allow them to make queries?

Comment (marcosfrm) [reply config/services/llmnr.xml:6]: I stand against any rule with `source-port` being enabled indirectly.

File: config/services/llmnr-client.xml
Comment (erig0) [inline config/services/llmnr-client.xml:5]: This is a duplicate description.

**Review by erig0 — CHANGES_REQUESTED** [review]
One change needed, otherwise lgtm.

**Review by erig0 — APPROVED** [review]
I cleaned up this commit and split it into two. Will merge after CI completes.

**Comment by yrro** [comment]
Ah thanks :)

**Pull request merged**
